app/services/trainees/create_from_hesa.rb

```diff
@@ -29,8 +29,9 @@ def call
         if trainee.save!
           create_degrees!
           create_placements!
+          store_application_choice_id!
```

Comment:
Just need to follow up on 
1. How Hesa can have an `Apply application id`
2. Hesa 6 digits vs apply 10 characters, are we actually referring to the correct item?
![image](https://github.com/DFE-Digital/register-trainee-teachers/assets/470137/8d6521a5-0f3e-46a8-8b88-dbcffb348393)

Reply:
@claire-hughez is this anything we need to consider on Register side?